packages/esm-patient-orders-app/src/lab-results/lab-results.resource.ts

```diff
@@ -142,15 +136,15 @@ const editObservation = async (obsUuid: string, obsPayload: any, abortController
   return editResultObs.status;
 };
 
-//TODO: the calls to update order and observations for results should be transactional to allow for rollback
+// TODO: the calls to update order and observations for results should be transactional to allow for rollback
 export async function updateOrderResult(
   orderUuid: string,
   encounterUuid: string,
   obsUuid: string,
   obsPayload: any,
   fulfillerPayload: any,
 ) {
-  const abortController = useAbortController();
+  const abortController = new AbortController();
```

Comment:
@denniskigen should we instead consider parameterising the AbortController? It's usages seems redundant here.

Reply:
Sure, @samuelmale, I've pushed a commit fixing that.